kalkulator.py

```diff
@@ -5,9 +5,11 @@ def dodaj(a,b):
     wynik=a+b
     return wynik
 
+def odejmij(a, b):
+    return a-b
```

Comment:
Moim zdaniem tutaj lepiej rozbic to na dwie linijki wprowadzając zmienną "wynik"